.github/workflows/ci.yml

```diff
@@ -25,17 +25,3 @@ jobs:
     secrets: "inherit"
```

Comment:
Not sure about your comment:

> This works locally, but I don't know what CI runs.

The run commands are from the line just above this comment (github does not allow me to comment on the right line). This "expand" to:

https://github.com/puppetlabs/cat-github-actions/blob/main/.github/workflows/gem_ci.yml


Overall :100: :star_struck: for cucumber AFAIAC!